targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L475xG/TARGET_MTB_STM_L475/PeripheralNames.h

```diff
@@ -0,0 +1,96 @@
+/* mbed Microcontroller Library
+
+* SPDX-License-Identifier: Apache-2.0
```

Comment:
this should be BSD-3-Clause instead of Apache 2.0 (see https://spdx.org/licenses/BSD-3-Clause.html)

Reply:
Thanks @0xc0170 .. Done!